### ISSUE TITLE
app crash when send to background

### ISSUE THREAD

**Issue by anastr** [stepstone-tech/android-material-app-rating#12]
Hi, the library crash when application detached from window.

throw `NotSerializableException`

```log
java.lang.RuntimeException: Parcelable encountered IOException writing serializable object (name = com.stepstone.apprating.AppRatingDialog$Builder$Data)

Caused by: java.io.NotSerializableException: com.stepstone.apprating.listener.OnNegativeButtonClickedListener$Companion$NULL$1
```
thanks

**Comment by zawadz88** [comment]
Hi,
Thanks for reporting this. We'll investigate :)

**Comment by pglebocki** [comment]
@anastr thanks for reporting.
Fix will be in version 1.3

**Comment by anastr** [comment]
thanks for the work.
When will release the next version?

**Comment by pglebocki** [comment]
We released version 2.0.

**Comment by anastr** [comment]
__Big thanks .__
